src/utils/cli.h

```diff
@@ -40,15 +51,36 @@ namespace Cli
         std::vector<std::string> extractFlag();
 
         /**
-         * @return Arguments specified on the commandline of this flag
+         * @param Index of the argument to get
+         * @return Arguments specified on the commandline of this flag. Empty string if invalid index.
+         */
+        const std::string& getArgs(size_t i);
```

Comment:
This function did something else before I implemented config-loading. Will change!